docs/project/glossary.md

```diff
@@ -232,34 +127,58 @@ officially updated since .NET Framework 2.0.
 
 ### Mono
 
-Mono is an open source alternative to the .NET Framework. Mono started around
+[Mono][mono] is an open source alternative to the .NET Framework. Mono started around
 the same time the .NET Framework was first released. Since Microsoft didn't
-release Rotor as open source, Mono was forced to start from scratch and is thus
+release Rotor as open source, Mono was forced to start from scratch and was thus
 a complete re-implementation of the .NET Framework with no shared code.
 
-When .NET Core was released under the MIT license, Microsoft also released large
-chunks of the .NET Framework under the MIT license as well, which can be found
-[here][referencesource]. This enabled the Mono community to use the same code
-the .NET Framework uses in order to close gaps and avoid behavioral differences.
+Today, the [Mono runtime](https://github.com/dotnet/runtime/tree/master/src/mono) is part
+of unified .NET platform. It is optimized for mobile (e.g. Xamarin) and browser (e.g. Blazor) scenarios.
+
+### CoreCLR
+
+Originally, CoreCLR was the runtime of Silverlight and was designed to run on multiple
+platforms, specifically Windows and OS X.
```

Comment:
```suggestion
platforms, specifically Windows and macOS.
```

Reply:
The platform name during Silverlight days was OS X. I am tempted to leave it as is.

Reply:
I guess it's talking history. That is a fair point then.